benchmarks/guest/ecrecover/src/main.rs

```diff
@@ -1,7 +1,10 @@
 use alloy_primitives::{Bytes, B256, B512};
-use k256::ecdsa::{Error, RecoveryId, Signature, VerifyingKey};
+// Be careful not to import k256::ecdsa::{Signature, VerifyingKey}
+// because those are type aliases that their (non-zkvm) implementations
+use ecdsa::{Signature, VerifyingKey};
+use k256::ecdsa::{Error, RecoveryId};
 use openvm::io::read_vec;
-use openvm_k256::Secp256k1Point;
+use openvm_k256::{Secp256k1, Secp256k1Point};
```

Comment:
I was wondering if you could just not have the `openvm-k256` import at all here, and instead in Cargo.toml patch `k256` with `openvm-k256`? that's how we would likely use it in practice